fuzzers/baby/baby_fuzzer_custom_executor/Cargo.toml

```diff
@@ -8,8 +8,9 @@ authors = [
 edition = "2021"
 
 [features]
-default = ["std"]
+default = ["std", "bloom_filter"]
```

Comment:
The feature flag should be called by the name of the feature, not by implementation detail. Maybe something like. "reexecution_filter" or similar?

Reply:
`bloom_input_filter` would be a middle ground(?)